stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 2.2.0
+version: 2.2.1
```

Comment:
This would be a major version bump as this could break people's current config

Reply:
@rjkernick - Do you have a version preference? Is it worth aligning with the Sonarqube version?

Reply:
A bump to 3.0.0 would be fine. Just needs to reflect the level of change to the chart.

Reply:
If we're changing the major version then I'd also suggest changing the default deployment strategy to `Recreate` as Sonarqube is a single node chart which has PVC issues when used with `RollingUpdate`.

Reply:
@rjkernick can we progress this?